ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -59,14 +59,12 @@ if(COMMAND ie_faster_build)
     )
 endif()
 
-target_link_libraries(onnx_importer PRIVATE onnx onnx_proto ${Protobuf_LIBRARIES} ngraph::builder
+target_link_libraries(onnx_importer PRIVATE onnx ngraph::builder
                                     PUBLIC ngraph)
 
 set(ONNX_INSTALL_INCLUDE "${NGRAPH_INSTALL_INCLUDE}/ngraph/frontend")
 target_include_directories(onnx_importer SYSTEM PUBLIC $<BUILD_INTERFACE:${ONNX_IMPORT_INCLUDE_DIR}>
                                                        $<INSTALL_INTERFACE:${ONNX_INSTALL_INCLUDE}>)
-target_include_directories(onnx_importer SYSTEM PRIVATE ${NGRAPH_INCLUDE_PATH}
```

Comment:
If I remember right we added ONNX headers as system in order to avoid some compiler warnings. Why did you remove it?

Reply:
I assumed that these props are inherited by linking to `onnx` and `ngraph` targets few lines above. But I overlooked that Protobuf_INCLUDE_DIRS is _privately_ added to INCLUDE_DIRECTORIES of `onnx_proto` target, and not to INTERFACE_SYSTEM_INCLUDE_DIRECTORIES.
I'll revert those changes.